src/Core/View.js

```diff
@@ -1112,6 +1132,35 @@ class View extends THREE.EventDispatcher {
             this.notifyChange(this.camera.camera3D);
         }
     }
+
+    /**
+     * Trigger underground visualization
+     * @param {boolean} [trigger] - Enable/Disable the underground visualization.
+     */
+    setUndergroundVisualization(trigger) {
+        const atmo = this.getLayerById('atmosphere');
+        if (trigger) {
+            this.tileLayer.updateTiledLayerOpacity({ target: this });
+            this.addEventListener(VIEW_EVENTS.CAMERA_MOVED,  this.tileLayer.updateTiledLayerOpacity);
```

Comment:
as now the code is the core, you can put it in the `tileLayer.preUpdate` and not use `addEventListener`